etc/docker/kernel-tf-gpu-py/Dockerfile

```diff
@@ -1,6 +1,17 @@
 # Ubuntu:xenial
 FROM tensorflow/tensorflow:1.12.0-gpu-py3
 
+RUN apt-get update && apt-get install -yq \
+    build-essential \
+    libsm6 \
+    libxext-dev \
+    libxrender1 \
+    netcat \
+    python-dev \
```

Comment:
This seems to be setting up Python 2.7 and thus failing the kernel start with `ImportError: No module named Crypto.Cipher`. Changing it to the same version of the python available in the image seems to solve the problem (e.g. `python3.5-dev`)

Reply:
@esevan BTW, please let me know if you want me to handle the update and I can fix when I merge this.

Reply:
@Iresende hmm.. That is strange. I found the list of those packages from jupyter/minimal-notebook.

> please let me know if you want me to handle the update and I can fix when I merge this.

Why not? Go ahead whenever you want! Thanks!

Reply:
I found it `/usr/bin/python -> python2.7` in the docker container. 
In jupyter notebook, `from Crypto import Cipher` works since jupyter notebook run on `python3`, but `bootstrap_kernel.sh` runs on `python`.
https://github.com/jupyter/enterprise_gateway/blob/461f3d3b3537a969f2dabd44278de2e7d3119a76/etc/kernel-launchers/bootstrap/bootstrap-kernel.sh#L15

This raised the error you pointed out.
`python2.7` is installed during dependency installation of `python-dev`, so that I replaced `python-dev` to `python3-dev` and checked `from Crypto import Cipher` works.
```
jovyan@esevan-desktop:/bootstrap$ ./bootstrap-kernel.sh 
./bootstrap-kernel.sh env: EG_RESPONSE_ADDRESS=localhost LD_LIBRARY_PATH=/usr/local/cuda/extras/CUPTI/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64 NVIDIA_VISIBLE_DEVICES=all NVIDIA_DRIVER_CAPABILITIES=compute,utility PATH=/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin PWD=/bootstrap KERNEL_ID=1234 KERNEL_LANGUAGE=python CUDA_PKG_VERSION=9-0=9.0.176-1 CUDA_VERSION=9.0.176 HOME=/home/jovyan SHLVL=2 _=/usr/bin/env
+ python /usr/local/bin/kernel-launchers/python/scripts/launch_ipykernel.py --RemoteProcessProxy.kernel-id 1234 --RemoteProcessProxy.response-address localhost --RemoteProcessProxy.spark-context-initialization-mode none
[D 2019-04-20 11:39:53,435.435 launch_ipykernel] Using connection file '/tmp/kernel-1234_362hf8x2.json'.
[E 2019-04-20 11:39:53,435.435 launch_ipykernel] Invalid format for response address 'localhost'. Assuming 'pull' mode...
```

I apologize that I can't check end-to-end test manually for EG since I work on things out of office, and I'm recently building up the working environment at home (Takes quite long time...). So please feel free to fix any update, thanks!